src/index.md

```diff
@@ -1723,7 +1724,7 @@ This list is intended to make it easier to identify which email list to include
 * `jdk.jdeps` – Core Libs
 * `jdk.jdi` – Serviceability
 * `jdk.jdwp.agent` – Serviceability
-* `jdk.jfr` – Runtime
+* `jdk.jfr` – JFR, Runtime
```

Comment:
Same as above, just JFR should be enough.

Reply:
Changed.